specification/components/schemas/responses/bookOrDeferAppointment/422Response.yaml

```diff
@@ -0,0 +1,32 @@
+description: |
+  Where status code 422 (Unprocessable Entity) is returned then an eRS-OperationOutcome-1 will be included in the body, as detailed below. 
+  Check diagnostics property for specific information regarding the error.
+  
+  | Error code                         | Description                                                                                                                  |
+  | ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+  | INVALID_VALUE                      | The input provided does not conform to the expected data types and format. |
+  | REFERENCE_NOT_FOUND                | A supplied reference could not be resolved to valid resource (e.g. a patient, clinician or an organisation). |
+  | NO_RELATIONSHIP                    | The user does not have a legitimate relationship with the request.    |
```

Comment:
For appointment status, we return INVALID_CODE if it is not one of the enumeration, but we also return INVALID_VALUE if it is not either 'booked' or 'proposed'

Reply:
I've added INVALID_CODE  to the list now, good catch :)